tests/test_api_endpoints.py

```diff
@@ -178,6 +180,28 @@ def test_system_rating(
         assert test_host_detail.json["rating"] == data_dict['rating']
 
 
+def test_executive_report(
+        auth_token,
+        db_setup,
+        db_create_account,
+        db_create_system,
+        db_create_performance_profile
+):
+    with app.test_client() as client:
+        response = client.get(
+            '/api/ros/v1/executive_report',
+            headers={"x-rh-identity": auth_token}
+        )
+        with app.app_context():
+            g = db_get_host('ee0b9978-fe1b-4191-8408-cbadbd47f7a3')
+            pprint(g.__dict__)
+        assert response.json['systems_per_state']['idling']['count'] == 1
+        assert response.json['systems_per_state']['idling']['percentage'] == 100
+        assert response.json['conditions']['cpu']['count'] == 2
+        assert response.json['conditions']['io']['count'] == 1
+        assert response.json['conditions']['memory']['count'] == 2
+
```

Comment:
@saltgen what you think about including a test case which will have all the data and also wanted to know in case the state or conditions information is not available will backend return the key with some value of the key will not be included in the response?

Reply:
@PreetiW The current test asserts the same as per test system. Also all keys will be returned irrespective of value.

Reply:
@PreetiW The current test asserts the same as per test system. Also all keys will be returned irrespective of value.